test/e2e/specs/beforesuite.go

```diff
@@ -0,0 +1,29 @@
+package specs
+
+import (
+	. "github.com/onsi/ginkgo"
+
+	"github.com/openshift/openshift-azure/test/clients/azure"
+	"github.com/openshift/openshift-azure/test/util/log"
+)
+
+var _ = BeforeSuite(func() {
```

Comment:
@Makdaam this would be better in an init() function in test/clients/azure/client.go - it's a bit spooky to declare a global in one place and populate it in another.

```golang
func init() {
	var err error

	RealRPClient, err = NewClientFromEnvironment(false, true)
	if err != nil {
		panic(err)
	}

	FakeRPClient, err = NewClientFromEnvironment(false, false)
	if err != nil {
		panic(err)
	}
}
```

Reply:
Moved the initialization to azure package. But I left the interpretation of ginkgo focus here since this is run before every test suite. This way the azure package contains all initialization code, and the ginkgo spec decides which client is needed.